content/spin/kv-store-tutorial.md

```diff
@@ -535,7 +517,7 @@ As we can see above, there is currently no data found at the `/test` endpoint of
 
 ## Conclusion
 
-We want to get feedback on the ergonomics of the key/value API. We are curious about what new APIs you would suggest we implement and are also interested in learning about what backing stores you would like to see. In the next iterations for this feature, we will focus on configuring multiple KV stores with multiple backing services (such as cloud services).
+We want to get feedback on the ergonomics of the key value API. We are curious about what new APIs you would suggest we implement and are also interested in learning about what backing stores you would like to see. In the next iterations for this feature, we will focus on configuring multiple KV stores with multiple backing services (such as cloud services).
```

Comment:
Remove last sentence: "the next iterations" will be a moving target and doesn't really belong in a tutorial.  We could have a "roadmap" section in the API guide if we want a place to discuss future plans?